src/providers/hana/qgshanaprovider.cpp

```diff
@@ -1665,6 +1648,12 @@ QgsVectorLayerExporter::ExportError QgsHanaProvider::createEmptyLayer(
   return QgsVectorLayerExporter::NoError;
 }
 
+QgsHanaProviderMetadata::QgsHanaProviderMetadata()
+  : QgsProviderMetadata( QgsHanaProvider::HANA_KEY, QgsHanaProvider::HANA_DESCRIPTION )
+{
+  QgsDebugMsg( QStringLiteral( "Create QgsHanaProviderMetadata" ) );
```

Comment:
please always use QgsDebugMsgLevel unless it's a critical debug message.

Reply:
Thank you for the hint.
Done in 546f3c2bf135.